spec/lib/user_segments_spec.rb

```diff
@@ -0,0 +1,176 @@
+require 'rails_helper'
+
+describe UserSegments do
+
+  describe "#all_users" do
+    it "returns all active users with newsletter enabled" do
+      active_user1 = create(:user, newsletter: true)
+      active_user2 = create(:user, newsletter: true)
+      active_user3 = create(:user, newsletter: false)
+      erased_user  = create(:user, erased_at: Time.current)
+
+      expect(UserSegments.all_users).to include active_user1
+      expect(UserSegments.all_users).to include active_user2
+      expect(UserSegments.all_users).to_not include active_user3
```

Comment:
Another Rubocop-rspec rule we have enabled (as you can see there are no issues for it on master branch) is [RSpec/NotToNot](http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/NotToNot) as for me " to not" is not something you would say in a normal conversation... "not to ..." is the normal form used. Also the fact that it differs from `.to` makes it easier for an eye to catch on that difference IMHO but we can discuss about it and change the rule enforcing about it in an issue if you want to